README.md

```diff
@@ -78,6 +78,21 @@ source ~/.bashrc
 ```
 > ***Note:*** The ROS2 daemon may need to be restarted to use the CycloneDDS RMW. Use `ros2 daemon stop` and `ros2 daemon start` to restart the daemon.
 
+### ROS 2 navigation stack
+Install the required ROS2 navigation dependencies:
+```bash
+sudo apt install ros-${ROS_DISTRO}-navigation2
+```
+
+### Additional dependencies
+Make sure the following tools and libraries are installed on your system:
+```bash
+sudo apt install python3-rosdep2 ninja-build screen
```

Comment:
I believe some of the packages can be moved to the respective sections, e.g. `python3-rosdep2` can be placed next to `python3-colcon-common-extensions python3-vcstool`. `screen` is required only for `DemoLevel2` and it might be moved there, etc.